src/app/shared/components/language-select/language-select.component.ts

```diff
@@ -6,9 +6,24 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./language-select.component.scss']
 })
 export class LanguageSelectComponent implements OnInit {
+  ru = 'ru';
+  eng = 'eng';
```

Comment:
I don't know exactly but I would create a mock interface with languages.

Reply:
Ok. I will create a task to refactor this component using mock when time is come.